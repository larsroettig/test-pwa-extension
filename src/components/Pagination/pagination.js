import React from 'react';
import {mergeClasses} from '@magento/venia-ui/lib/classify';
import defaultClasses from './pagination.css';
import { array, number } from 'prop-types';

const Pagination = props => {
    const classes = mergeClasses(defaultClasses, props.classes);

    return (
        <div className={classes.root}>
            <h1>skkdwkwke</h1>
        </div>
    )
}

Pagination.propTypes = {
    pages: array,
    currentPage: number,
}

export default Pagination;