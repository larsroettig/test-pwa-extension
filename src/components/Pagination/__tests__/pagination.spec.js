import React from 'react';
import Pagination from '../pagination';
import { render } from '@testing-library/react';

jest.mock('@magento/venia-ui/lib/classify');

test('renders correctly', () => {

    const { asFragment } = render(<Pagination pages={[]} currentPage={1} />);
    expect(asFragment()).toMatchSnapshot();
});