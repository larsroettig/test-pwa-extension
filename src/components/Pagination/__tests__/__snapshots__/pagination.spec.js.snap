// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<DocumentFragment>
  <div
    class="root"
  >
    <h1>
      skkdwkwke
    </h1>
  </div>
</DocumentFragment>
`;
